WORKSPACE

```diff
@@ -40,12 +40,19 @@ load(
 
 container_go_deps()
 
+load(
```

Comment:
add this to top level README in ## Setup section

Reply:
Done